doc/null_safety/null_safe_migration.md

```diff
@@ -16,29 +17,76 @@ If you haven't already, familiarize yourself with the concepts of Dart null safe
 - Dart's [Understanding null safety](https://dart.dev/null-safety/understanding-null-safety) article 
 - OverReact's documentation on [null safety and required props](../null_safety_and_required_props.md)
 
-## Step 2: Run the preparation codemod (coming soon!)
+## Step 2: Run the preparation codemods
 
-<!-- TODO - update this section once we release the `null_safety_prep` codemod. -->
-
-> [!WARNING]
-> This codemod does not exist yet, but is coming soon!
-
-Start out by running the `null_safety_prep` [codemod][orcm] on your repo:
+Start out by running the [codemod][orcm]s on your repo:
 
 ```
 dart pub global activate over_react_codemod
-dart pub global run over_react_codemod:null_safety_prep
+dart pub global run over_react_codemod:null_safety_prep --yes-to-all
+dart pub global run over_react_codemod:null_safety_migrator_companion --yes-to-all
 ```
 
-This codemod will:
+These codemods will:
 - Migrate as many over_react specific use cases as possible.
 - Get the repo into a state where the migration tool can be run with less manual intervention.
-
-## Step 3: Run the migration tool
+- Add nullability hints to props/state that are defaulted/initialized in class components.
+  - These hints will cause defaulted/initialized values to be migrated as "late required". 
+    See our [prop nullability](#prop-nullability) docs for more details on whether you should keep them required following the migration.
+
+## Step 3: Enable the analyzer plugin
+
+Enabling the analyzer plugin will help you spot potential null-safety issues in your components at analysis time,
+especially as it pertains to [prop nullability](#prop-nullability).
+
+1. Enable the plugin in your `analysis_options.yaml` file:
+    ```yaml
+    analyzer:
+      plugins:
+        - over_react
+    ```
+1. Restart the Dart Analysis Server in your IDE. The plugin may take a minute to load after built-in analysis completes.
+
+   * If the number of new lints / errors is overwhelming, you an also use this snippet to disable all lints except for the ones that are particularly useful during a null-safety migration:
+       ```yaml
+       analyzer:
+         errors:
```

Comment:
Does this work? I thought analyzer plugin lints couldn't be disabled this way, and needed the plugin-specific configuration [here](https://github.com/Workiva/over_react/tree/master/tools/analyzer_plugin#configuration).

Reply:
I think you're right

Reply:
https://github.com/Workiva/over_react/pull/932/commits/0adf86376d1557a2e33cdbcbcbf2c70fbc0af418